apps/accounts/models.py

```diff
@@ -98,6 +99,16 @@ def should_updated_access_token(self):
         spotify_auth_timeout = timezone.now() - timedelta(seconds=settings.SPOTIFY['auth_user_token_timeout'])
         return self.last_refreshed < spotify_auth_timeout
 
+    def refresh_access_token(self):
+        """Make a call to the Spotify API to refresh the access token for the SpotifyUserAuth records"""
+        spotfiy = SpotifyClient(identifier='update-access-token:{}'.format(self.user.username))
```

Comment:
Add handling for errors raised from the Spotify client. What do we do if the API is down?

Reply:
If API is down, log a warning and reraise the error. If we can't update the access tokens we can't do anything with Spotify on the users behalf anyway